### ISSUE TITLE
make OpenStreetMap API endpoint configuration

### ISSUE THREAD

**Issue by mtmail** [kepta/idly#2]
Default endpoint seems to be `https://www.openstreetmap.org/api/0.6/map?bbox=${bboxStr}` (`src/helpers/helpers.ts`).

https://wiki.openstreetmap.org/wiki/API_v0.6#When_NOT_to_use_the_API reads "The API is primarily intended for editing. For read-only purposes or projects, see API usage policy." https://operations.osmfoundation.org/policies/api/

**Comment by gravitystorm** [comment]
Yes, unfortunately while this is cool technology demo, it directly violates the [API Usage Policy](https://operations.osmfoundation.org/policies/api/).

**Comment by kepta** [comment]
Since it is a developer tool and not directly for users to use, I will make the API URL configurable and let the developers decide what API to use in case they violate  [API Usage Policy](https://operations.osmfoundation.org/policies/api/)

**Comment by mmd-osm** [comment]
You could try one of the public Overpass instances as well, as they offer the same map call, e.g.

https://overpass-api.de/api/map?bbox=9.18143,45.46142,9.20514,45.47089

See https://wiki.openstreetmap.org/wiki/Overpass_API for a list of available instances.

kumi.systems Overpass API instance might be a good start for your project.
